.gitignore

```diff
@@ -1,4 +1,5 @@
 *.swp
+*.swo
```

Comment:
@birdage Is this something we need/want.  This is the only remnant from Dan's PR